src/main/java/fi/metatavu/dcfb/server/search/handlers/AbstractIndexableHandler.java

```diff
@@ -63,6 +65,27 @@ public void deleteIndexable(UUID id) {
    */
   protected abstract String getType();
 
+  /**
+   * Creates GeoPoint for a Location object
+   * 
+   * @param location location
+   * @return GeoPoint
+   */
+  protected GeoPoint createGeoPoint(Location location) {
+    if (location == null) {
+      return null;
+    }
+    
+    BigDecimal latitude = location.getLatitude();
+    BigDecimal longitude = location.getLongitude();
+    
+    if (longitude == null || longitude == null) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this expression which always evaluates to "false" ([squid:S2589](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS2589))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=21&issues=AWVnq7t2XKENQn1Shuhq&open=AWVnq7t2XKENQn1Shuhq)

[ISSUE_KEY]: AWVnq7t2XKENQn1Shuhq